crates/turborepo-repository/src/package_graph/mod.rs

```diff
@@ -30,6 +30,21 @@ pub struct PackageGraph {
     lockfile: Option<Box<dyn Lockfile>>,
 }
 
+#[derive(Eq, PartialEq, Hash)]
+pub struct WorkspacePackage {
+    pub name: WorkspaceName,
+    pub path: String,
```

Comment:
I want this to be `AnchoredSystemPath` or `AnchoredSystemPathBuf` (not sure which), but when I do that Rust complains when I try to use it with `HashSet<WorkspacePackage>` because the size of `[u8]` is not known at compile time

Reply:
Looks like I get the same compile error even for `Vec<WorkspacePackage>`:

```
error[E0277]: the size for values of type `[u8]` cannot be known at compilation time
   --> crates/turborepo-repository/src/package_graph/mod.rs:328:17
    |
328 |     ) -> Result<Vec<WorkspacePackage>, ChangedPackagesError> {
    |                 ^^^^^^^^^^^^^^^^^^^^^ doesn't have a size known at compile-time
    |
```

Reply:
You'll want `AbsoluteSystemPathBuf` as `AnchoredSystemPath` is itself a [dynamically sized type](https://doc.rust-lang.org/reference/dynamically-sized-types.html)

Reply:
thank you!